subsys/nrf_rpc/include/nrf_rpc_utils.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef UTILS_NRF_RPC_UTILS_H_
+#define UTILS_NRF_RPC_UTILS_H_
+
+const char *get_version();
```

Comment:
The function should have correct naming with prefix. Are you going to provide random parameters? If so, it should be documented which ones.

Reply:
applied nrf_rpc prefix.